index.html

```diff
@@ -14,11 +14,36 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="header">
+      <h3>Code Journel</h3>
```

Comment:
You misspelled "journal". Also, you should have elements with the `container`, `row`, and `column` classes in here.